lib/int3.h

```diff
@@ -161,10 +161,12 @@ class int3
 	std::string toString() const
 	{
 		//Performance is important here
-		char str[16] = {};
-		std::sprintf(str, "(%d %d %d)", x, y, z);
+		std::string result = "{" +
+				std::to_string(x) + " " +
+				std::to_string(y) + " " +
+				std::to_string(z) + "}";

```

Comment:
No impact on performance?
Will change of ( with { not brake anything?

Reply:
Performance - old code was already performing relatively expensive operation of constructing std::string (see return type), so no. "{" vs "()" - my bad, will fix